controller/products.js

```diff
@@ -0,0 +1,29 @@
+
+const Products=require("../data/Products.json")
+
+exports.getAllProduct=async(req,res)=>{
+    try{
+
+        // const {minPrice,maxPrice}=req.body
```

Comment:
min max price filtering issue isn't yet resolved remove that code. Pull the latest commit and don't include additional lines of code just copy paste the code by making right directories

Reply:
is the code is not working for min max price ??

you mean I need to remove the commented line from the code??

Reply:
yes your PR is still under review for min max price product don't include that code